src/layouts/MainLayout.tsx

```diff
@@ -2,24 +2,24 @@ import { Box, CssBaseline, Divider, IconButton, List } from "@mui/material";
 import MuiDrawer from "@mui/material/Drawer";
 import { CSSObject, Theme, styled } from "@mui/material/styles";
 import { useState } from "react";
-import ChevronLeftIcon from "@mui/icons-material/ChevronLeft";
-import ChevronRightIcon from "@mui/icons-material/ChevronRight";
-import MailIcon from "@mui/icons-material/Mail";
-import InboxIcon from "@mui/icons-material/MoveToInbox";
+import { CaretLeft } from "@phosphor-icons/react";
+import { CaretRight } from "@phosphor-icons/react";
+import { EnvelopeSimple } from "@phosphor-icons/react";
+import { ArchiveTray } from "@phosphor-icons/react";
 import ListItem from "@mui/material/ListItem";
 import ListItemButton from "@mui/material/ListItemButton";
 import ListItemIcon from "@mui/material/ListItemIcon";
 import ListItemText from "@mui/material/ListItemText";
-import HouseOutlinedIcon from "@mui/icons-material/HouseOutlined";
-import AssignmentTurnedInOutlinedIcon from "@mui/icons-material/AssignmentTurnedInOutlined";
+import { House } from "@phosphor-icons/react";
+import { NotePencil } from "@phosphor-icons/react";
```

Comment:
Could you add all imported icons like this

import {
  ArchiveTray,
  CaretRight,
  EnvelopeSimple,
  NotePencil,
  CaretLeft,
  House,
  CurrencyCircleDollar,
  Notepad,
  User,
} from "@phosphor-icons/react";

?